apps/src/templates/sectionProgressV2/LevelDataCell.jsx

```diff
@@ -149,6 +151,30 @@ function LevelDataCell({
       </Link>
     </td>
   );
+
+  if (metadataExpanded) {
+    return (
+      <div className={styles.lessonDataCellExpanded}>
+        {levelCellUnexpanded}
+        <div
+          className={classNames(styles.gridBox, styles.gridBoxMetadata, {
+            [styles.gridBoxChoiceSubLevel]: level.parentLevelId !== undefined,
+          })}
+        >
+          {!level.parentLevelId && formatTimeSpent(studentLevelProgress)}
+        </div>
+        <div
+          className={classNames(styles.gridBox, styles.gridBoxMetadata, {
+            [styles.gridBoxChoiceSubLevel]: level.parentLevelId !== undefined,
```

Comment:
I went back and forth on the right way to modify this for choice levels.  I decided to keep it all in this return statement rather than create a separate function that would slightly modify the return depending on if the cell is for a choice level or not.  Open to feedback on that.

Reply:
Yeah it's more of an art than a science to decide when there's enough shared logic to keep two code paths together vs. splitting them up. I think you did well here.